gitbook-docs/market/cmd-reference/proof.md

```diff
@@ -2,60 +2,54 @@
 description: Scripts to manage submission/retrieval of proofs
 ---
 
-# proof
+# Proof
 
-## Submit Proof
+## Submit proof
 
-```
-python proof_tools.py push --h
-usage: proof_tools.py push [-h] [-a ASK_KEY] [-b BID_KEY] -f FILE
+To submit proof to the Proof Market, you'll need the proof file and request or proposal key.
+You can provide only one of the two possible keys.
+
+### Usage
+
+```console
+foo@bar:~$ python3 proof_tools.py push --h
+
+usage: proof_tools.py push [-h] [-a request_KEY] [-b proposal_KEY] -f FILE
 
 options:
   -h, --help            show this help message and exit
-  -a ASK_KEY, --ask_key ASK_KEY
-                        ask_key
-  -b BID_KEY, --bid_key BID_KEY
-                        bid_key
+  -a request_KEY, --request_key request_KEY
```

Comment:
I think this should have a consistent case. ie REQUEST_KEY. If not, the dev team should update the docs in python.

Reply:
You're right, that's my mistake. Fixed, thank you